tutor/test_t1_18_CreateAnExternalAssignment.py

```diff
@@ -327,8 +321,8 @@ def test_teacher_dates_for_periods_individually_8088(self):
                 today + datetime.timedelta(days=len(periods) + 5)
             ).strftime('%m/%d/%Y')
             element = self.teacher.find(
-                By.XPATH, '//div[contains(@class,"tasking-plan")' +
-                'and contains(@data-reactid,":' + str(x + 1) + '")]' +
+                By.XPATH, '//div[contains(@class,"tasking-plan")]' +
+                '[' + str(x + 1) + ']' +
```

Comment:
these should really be string formatted so this becomes
'//div[contains(@class,"tasking-plan")][%s]' % (x + 1) +
and breaks naturally around the path control